### DISCUSSION TITLE
support Apple ProRAW ?

### DISCUSSION THREAD

**Discussion by Ghisura** [immich-app/immich#57] [Q&A] [answered]
Hello,
I'm testing, and apparently Immich doesn't (yet?) support Apple ProRAW. 
Apparently the mobile application starts from the most recent file, if the first file is in jpg or heic the transfer takes place, but as soon as the file is in ProRAW format, the transfer freezes. 
Do you foresee future support for the ProRAW format?
Thanks.

**Comment by alextran1502** [comment, marked as answer]
Thank you for your suggestion. I honestly just know about this format from you 😃 if it is from Apple, I will find the way to support it!

**Comment by Ghisura** [comment]
Hello,
Just to confirm, that the export of ProRAW work great, now 👍 
Thanks a lot

**Reply by alextran1502** [reply]
Yep, no problem! Forgot to tell you when the new version got released 😅
